harbor-cluster/values.yaml

```diff
@@ -7,8 +39,21 @@ core:
     class: nginx
     issuer:
       create: false
-      kind: ClusterIssuer
+      kind: Issuer #ClusterIssuer
       name: letsencrypt-production-http
+      acme:
+        email: tarik.haddouchi@advatys.com
+        server: https://acme-v02.api.letsencrypt.org/directory
+  oidc:
+    enabled: true
+    name: "keycloak"
+    endpoint: "http://keycloak-keycloak.apps.sandbox.thadvatys.com/auth/realms/harbor"
```

Comment:
the endpoint should be: http://oidcsubdomain.example.org/auth/realms/harbor"

Reply:
ok